src/store.ts

```diff
@@ -43,6 +48,13 @@ const store: StoreOptions<RootState> = {
                 commit('initKeys', keys);
             });
         },
+        setKeyguardResult({ commit }, result) {
+            commit('setKeyguardResult', result);
+            commit('setWaitingForKeyguard', false);
```

Comment:
Is this necessary? I think the state variable is always initialized as `false` when the page is loaded (which is the case, when returning from the keyguard).

Also, this is just two commits without any other actions performed, so could also be a mutation directly, I'd say?